system-test/managed_writer_client_test.ts

```diff
@@ -554,6 +554,105 @@ describe('managedwriter.WriterClient', () => {
     }).timeout(30 * 1000);
   });
 
+  it('should fill default values when MissingValuesInterpretation is set', async () => {
+    bqWriteClient.initialize();
+    const client = new WriterClient();
+    client.setClient(bqWriteClient);
+
+    const updatedSchema: TableSchema = {
+      fields: [
+        ...(schema.fields || []),
+        {
+          name: 'id',
+          type: 'STRING',
+          defaultValueExpression: 'GENERATE_UUID()',
+        },
+        {
+          name: 'created_at',
+          type: 'TIMESTAMP',
+          defaultValueExpression: 'CURRENT_TIMESTAMP()',
+        },
+        {
+          name: 'updated_at',
+          type: 'TIMESTAMP',
+          defaultValueExpression: 'CURRENT_TIMESTAMP()',
+        },
+      ],
+    };
+    const [table] = await bigquery
+      .dataset(datasetId)
+      .createTable(tableId + '_default_values', {schema: updatedSchema});
+    const parent = `projects/${projectId}/datasets/${datasetId}/tables/${table.id}`;
+
+    const storageSchema =
+      adapt.convertBigQuerySchemaToStorageTableSchema(updatedSchema);
+    const protoDescriptor: DescriptorProto =
+      adapt.convertStorageSchemaToProto2Descriptor(storageSchema, 'root');
+
+    // Row 1
```

Comment:
These sample rows are confusing, as the row data isn't compatible with the `updatedSchema` above.  If you're wanting empty rows that's fine, just describe what you're doing so the interaction is clearer.

Reply:
I've pushed some updates to the test to make the full schema more explicit. The idea here is to send rows with some empty values and show that they are being filled or not accordingly to the MVI settings.